### ISSUE TITLE
[Horizon] Button/Input in Toolbar: Missing top border while button/input is focused

### ISSUE THREAD

**Issue by C3ntraX** [SAP/ui5-webcomponents#5336]
### **Bug Description**
Missing top border while button or input is in focused.
The Button/Input is used inside a toolbar (44px by default).

Theme: Horizon

### **Expected Behavior**
A border around the button, if the button is focused.

### **Steps to Reproduce**
2. ...
3. ...

### **Isolated Example**
If possible, please provide an isolated example by forking [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html).

### **Context**
- UI5 Web Components version: {...}
- OS/Platform: {...}
- Browser: {...}
- Affected component: {...}

### **Log Output / Stack Trace / Screenshots**
![image](https://user-images.githubusercontent.com/44802503/172403882-6d026075-b14e-4278-b55a-d99cf66f17a4.png)
![image](https://user-images.githubusercontent.com/44802503/172404654-3c5ea93b-da07-4413-9ca5-a9fe2acc6c52.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by niyap** [comment]
Hello @C3ntraX, 

I am not sure whether I understand you right. I am not able to check the issue in the provided CodeSandbox, as it could not be loaded in Horizon. Here, you can check how to make it works with Horizon: https://sap.github.io/ui5-webcomponents/playground/getting-started/using-assets/
On the other hand, 52px is the expected height for the ShellBar in horizon.
There was similar [issue](https://github.com/SAP/ui5-webcomponents/pull/5277) which is already fixed, but I am not sure whether it covers your use case as well, as I am not able to check the reproducible issue.

Please adjust your CodeSandbox in order to be able to reproduce the issue and retest on master whether the issue is still reproducible, after the mentioned change.

Kind Regards,
Niya

**Comment by C3ntraX** [comment]
Hello @niyap 

the codeexample is not important, I found my issue but I have a problem, let me explain:

let me try again:
The problem is, that different themes have different heights for the shellbar.
1. "sap_horizon" has 52px height
2. "sap_fiori_3" "sap_belize" have 44px height

In the example for react the ShellBar has display: "fixed" which means, we need an div for empty content (under the ShellBar)
I mean, yes I could set a global height if the themes changes to "sap_horizon" but this shouldn´t be necessary.

See your template for react:
https://github.com/SAP/ui5-webcomponents-react/blob/main/packages/cra-template-seed/template/src/components/Shell/Shell.js

`const style = {
  shell: {
    position: 'fixed',
    width: '100%',
    zIndex: 100,
  },
  emptySpace: {
    paddingTop: '44px',
  },
};`

`<div  style={style.emptySpace} />`

Now the problem:
I have a component where I can change the theme, but I have problems to react for this change in the style:
`emptySpace: {
    paddingTop: '44px',
  },`

This assumes the height is fixed with 44px but it isn´t.

Because of this problem, I ased for an enhancement:
1. Is there a way to add a property to the Shellbar like "static" so that I don´t need to set a fixed height for the shellbar?
2. Or can you add a ShellbarHeight css variable, so that I can set the variable instead of the fixed height?

**Comment by niyap** [comment]
Hello Colleagues,

As the issue is not in UI5 Web Components project, I am closing the incident.

Kind Regards,
Niya

**Comment by MarcusNotheis** [comment]
I'll pull this over to the UI5 WCR repo
